src/_includes/layouts/base.njk

```diff
@@ -28,6 +28,7 @@
           <li{% if '/blog' in page.url %} class="current"{% endif %}><a href="/blog">Blog</a></li>
           <li{% if '/docs' in page.url %} class="current"{% endif %}><a href="/docs">Docs</a></li>
           <li{% if '/features' in page.url %} class="current"{% endif %}><a href="/features" title="JavaScript and WebAssembly language features">JS/Wasm features</a></li>
+          <li{% if '/grant' in page.url %} class="current"{% endif %}><a href="/grant" title="Research">Research</a></li>
```

Comment:
```suggestion
          <li{% if '/grant' in page.url %} class="current"{% endif %}><a href="/grant">Research</a></li>
```

Reply:
Oops, thanks, I think I just copied&modified the line above.